tests/widget/inline/widget-inline.ca_fr.test.js

```diff
@@ -1,9 +1,7 @@
 import { expect, test } from '@playwright/test';
 
-test('inline widget in Canadian French', async ({ browser }) => {
-  const context = await browser.newContext({ locale: 'fr-CA' });
-  const page = await context.newPage();
-
+test.use({ locale: 'fr-CA' });
+test('inline widget in Canadian French', async ({ page }) => {
```

Comment:
I think this pattern is a bit more concise and keeps it consistent with the rest of the tests.